guides/Web_Frameworks/Phalcon.md

```diff
@@ -0,0 +1,26 @@
+# PHALCON:
+Phalcon is an open source,high performance and one of the most popular PHP web framework delivered as an extension of C but written in Zephir (its a compiler written in PHP to convert Zephir language into C extension).
+
+### WHY USE PHALCON?
+1.Faster Execution.
+
+2.Well designed features and framework.
+
+3.Can handle more HTTP requests per second than comparable frameworks written in PHP.
+
+4.Reduced resource usage.
+
+### PHALCON INSTALLATION:
+Phalcon needs PHP to run. Its loosely coupled design (system in which each of its components has, or makes use of,little or no knowledge of the definitions of other separate components) allows developers to install Phalcon and use its functionality without additional extensions.
+
+[INSTALLING PHALCON ON WINDOWS,LINUX,MAC](https://docs.phalconphp.com/zh/3.3/installation)
+
```

Comment:
What to add:
- Installation basic instructions for PHP and Phalcon
- Perhaps why Phalcon is more preferred by PHP devs( no bullet points, I want clear and concise explanations)